requirements.txt

```diff
@@ -1,3 +1,3 @@
 # basic
-setuptools==52.0.0
+setuptools==65.5.1
```

Comment:
The latest `setuptools` is already 67.2.0: https://pypi.org/project/setuptools/

We are already using the latest `pip`. Maybe we should unpin it altogether.

Reply:
This file only affects Mac user. I think it's okay to unpin it.
Linux packages do not use this file either.
```
/opt/az/bin/python3 -m pip list | grep setup
setuptools                              58.1.0
```

Reply:
This issue is caused by old `setuptools` version.  https://github.com/Azure/azure-cli/issues/25682